src/Constraint/DraftFour/Type.php

```diff
@@ -13,6 +13,26 @@
     const KEYWORD = 'type';
 
     /**
+     * Whether examples like 98249283749234923498293171823948729348710298301928331
+     * and "98249283749234923498293171823948729348710298301928331" are valid strings.
+     */
+    const BIGINT_MODE_STRING_VALID = 1;
```

Comment:
Boolean should do fine, indeed. And I think that is somewhat verbose too. Let us think of something for a while...